probe/websocket/ws_test.go

```diff
@@ -66,9 +74,8 @@ func TestWSPing(t *testing.T) {
 			Headers: test.Headers,
 		}
 		ws.Config(global.ProbeSettings{})
-		ok, _ := ws.DoProbe()
-		assert.Equalf(t, test.Want, ok, fmt.Sprintf("case %d", i))
-
+		ok, err := ws.DoProbe()
+		assert.Equalf(t, test.Want, ok, fmt.Sprintf("case %d %s", i, err))
```

Comment:
Consider using a more descriptive error message in the assertion. Instead of just printing the case number and error, you could include more context about what the test case is checking and why it might have failed. [medium]